ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/components/ClusterStatus.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Button, Popover } from '@patternfly/react-core';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+
+import { getClusterStatusObject } from '../compliance.scanConfigs.utils';
+
+export type ClusterStatusProps = {
+    errors: string[];
+};
+
+function ClusterStatus({ errors }: ClusterStatusProps) {
+    const statusObj = getClusterStatusObject(errors);
```

Comment:
A question for thought, not necessarily a change request: now that you encapsulated `ClusterStatus` component (bravo) what are pro and con to move the logic of `getClusterStatusObject` util function inline here?